config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   resources :books, only: [:index]
+  resources :authors, only: [:show]
+  get "/authors/:id/books", to: "authors#books"
```

Comment:
Thanks @dstrunk, good catch.  I'll refactor it out and make a commit showing the edit.